src/frontend/komponenter/Fagsak/Fakta/FeilutbetalingFaktaContext.tsx

```diff
@@ -191,6 +232,30 @@ const [FeilutbetalingFaktaProvider, useFeilutbetalingFakta] = createUseContext(
                     melding: feilmelding,
                 });
             }
+            if (vurderingAvBrukersUttalelse?.harBrukerUttaltSeg === HarBrukerUttaltSegValg.JA) {
+                const feilmeldingBeskrivelseBrukerHarUttaltSeg = _validerTekst3000(
+                    // @ts-ignore
+                    vurderingAvBrukersUttalelse?.beskrivelse
+                );
+                if (feilmeldingBeskrivelseBrukerHarUttaltSeg) {
+                    feilmeldinger.push({
+                        gjelderBeskrivelseBrukerHarUttaltSeg: true,
+                        gjelderBegrunnelse: false,
+                        melding: feilmeldingBeskrivelseBrukerHarUttaltSeg,
+                    });
+                }
+            }
+            if (
+                vurderingAvBrukersUttalelse?.harBrukerUttaltSeg !== HarBrukerUttaltSegValg.NEI &&
```

Comment:
Kunne man sjekket IKKE_VURDERT?

Reply:
Må vel isåfall sjekke både null og ikke_vurdert, men det er mulig